admin/resources/js/components/form/Checkbox.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import get from "lodash/get";
+import { RegisterOptions, useFormContext } from "react-hook-form";
+import { InputWrapper } from "../H2Components/InputWrapper";
+
+export interface CheckboxProps
+  extends Omit<React.InputHTMLAttributes<HTMLInputElement>, "type"> {
+  /** HTML id used to identify the element. */
+  id: string;
+  /** Holds text for the label associated with the input element */
+  label: string;
+  /** A string specifying a name for the input control. */
+  name: string;
+  /** Set of validation rules and error messages to impose on input. */
+  rules?: RegisterOptions;
+  /** Optional context which user can view by toggling a button. */
+  context?: string;
+}
+
+const Checkbox: React.FunctionComponent<CheckboxProps> = ({
+  id,
+  label,
+  name,
+  rules = {},
+  context,
+  ...rest
+}) => {
+  const {
+    register,
+    formState: { errors },
+  } = useFormContext();
+  // To grab errors in nested objects we need to use lodash's get helper.
+  const error = get(errors, name)?.message;
+
+  return (
+    <InputWrapper
+      inputId={id}
+      label={label}
+      required={!!rules.required}
+      context={context}
+      error={error}
+      hideOptional
+    >
+      <input
+        style={{ order: -1 }}
```

Comment:
What's the purpose of the css property `order` required here?

Reply:
It puts the input (the checkbox) to the left of the label.